tsl/test/expected/chunk_api-13.out

```diff
@@ -819,8 +821,113 @@ SELECT tablespace FROM pg_tables WHERE tablename = :'CHUNK_NAME';
  tablespace1
 (1 row)
 
+-- Now create the complete chunk from the chunk table
+SELECT _timescaledb_internal.create_chunk('chunkapi', :'SLICES', :'CHUNK_SCHEMA', :'CHUNK_NAME',
+	   format('%I.%I', :'CHUNK_SCHEMA', :'CHUNK_NAME')::regclass);
+                                              create_chunk                                               
+---------------------------------------------------------------------------------------------------------
+ (11,10,_timescaledb_internal,_hyper_10_10_chunk,r,"{""time"": [1514419200000000, 1515024000000000]}",t)
+(1 row)
+
+SELECT * FROM test.show_constraints(format('%I.%I', :'CHUNK_SCHEMA', :'CHUNK_NAME')::regclass);
+     Constraint      | Type | Columns | Index |              Expr              | Deferrable | Deferred | Validated 
+---------------------+------+---------+-------+--------------------------------+------------+----------+-----------
+ chunkapi_temp_check | c    | {temp}  | -     | (temp > (0)::double precision) | f          | f        | t
```

Comment:
Note that primary key and foreign key constraints are missing. This is intentional, since no enforcement is currently in place to ensure these constraints exist.